package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@signalk/client",
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
I prefer changing the version in a separate commit outside the pr. If this pr needs more work and something else is merged and published before this one then this needs changing. Or you may decide to wait for another change to only publish one version.